TestAssets/TestPackages/dotnet-invoke-portable/Program.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.IO;
+using Microsoft.DotNet.Cli.Utils;
+using NuGet.Frameworks;
+
+namespace ConsoleApplication
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            var environment = new EnvironmentProvider();
+            var packagedCommandSpecFactory = new PackagedCommandSpecFactory();
```

Comment:
Why are we not using `dotnet-dependency-tool-invoker`?  Doesn't this do the same thing?